packages/core/helper-plugin/src/components/tests/PageSizeURLQuery.test.tsx

```diff
@@ -60,7 +60,7 @@ describe('PageSizeURLQuery', () => {
 
   describe('interaction', () => {
     it('should change the value when the user selects a new value', async () => {
-      let testLocation = null;
+      let testLocation: { search?: string } = {};
```

Comment:
I think this can be typed better is you use something like `ReturnType<typeof useLocation>` to get the location :)